mindsdb/api/executor/sql_query/steps/subselect_step.py

```diff
@@ -56,7 +66,11 @@ def f_all_cols(node, **kwargs):
             def inject_values(node, **kwargs):
```

Comment:
I think is better to replace `inject_values` function and use `get_fill_param_fnc' function instead (which is used by other steps to fill parameter values):
```
fill_params = get_fill_param_fnc(self.steps_data)
query_traversal(query, fill_params)
```